tests/conftest.py

```diff
@@ -128,3 +130,27 @@ def default_10000_blocks_compact(bt):
 def tmp_dir():
     with tempfile.TemporaryDirectory() as folder:
         yield Path(folder)
+
+
+@pytest_asyncio.fixture(scope="function")
+async def wallet_node(self_hostname):
+    async for _ in setup_node_and_wallet(test_constants, self_hostname):
+        yield _
```

Comment:
Is this supposed to be added here?

Reply:
Not, I think that is a merge mistake on my part, thanks!